docs/HowTo/Configure/Genesis-File.md

```diff
@@ -1,20 +1,23 @@
-description: Configuring network using genesis file  
+description: Configuring a network using the genesis file
 <!--- END of page meta data -->
 
-# Creating Hyperledger Besu Genesis File 
+# Creating the Hyperledger Besu genesis file
 
-The genesis file defines the first block in the chain and the first block defines which chain you want to join.
+The genesis file defines the first block in the chain and the first block defines which chain you
+want to join.
 
-For Ethereum MainNet and public testnets (for example, Rinkeby), the genesis configuration is defined in 
-Besu and used when a public network is specified using the [`--network`](../../Reference/CLI/CLI-Syntax.md#network)
-command line option. 
+For Ethereum mainnet and public testnets (for example, Rinkeby) the genesis configuration
```

Comment:
MainNet is in our style guide

Reply:
Ah, OK. I've seen other instances of "mainnet" which I will change if I see them again.